apps/site/assets/ts/components/Badge.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+const Badge = ({ text }: { text: string }): JSX.Element => {
```

Comment:
praise: Appreciate this being broken out for reusability! 
suggestion: Is there a role that should be added for accessibility? Maybe the [alert](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/alert_role) role?

Reply:
I am a bit hesitant to use an alert role as it sounds like screen readers just say it, regardless of where the user is looking.  So if one route is suspended, they will just hear "Suspended" upon the page loading.  Also they would hear multiple if multiple badges are present.  This is what is says on the documentation about the alert role:
`Warning: Because of its intrusive nature, the alert role must be used sparingly and only in situations where the user's immediate attention is required.`

Reply:
Ah, good point! Maybe [status](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/status_role) would be more appropriate then?

Reply:
I am leaning towards we don't need to add more context.  I think the text precisely describes that status of the route.
I imagine the screen reader saying the visible text "Wonderland Suspended" or "Wellington Shuttle Service".  Under normal circumstances it would say "Wonderland 5min 15min".  That seems like enough context to put together what's happening.